README.md

```diff
@@ -15,40 +15,47 @@ pip install -r requirements.txt
 
 ## Getting started
 
-You can see the pipeline run on some sample data by running the following script:
+Spin up the main UI via:
 
 ```bash
-./sample.sh
+python src/app.py
 ```
 
+Then open http://localhost:5000 on your browser. There is sample data to play with
+in the `data/sample/` directory that can be used to test the code / UI.
+
 ## Usage
 
-Put your own exported credit card transactions in the `data/raw/` directory,
-the filenames need to respect the following conventions:
-- American Express files must start with `amex`
-- Capital One files must start with `capital`
-- Chase files must start with `chase`
-- USBank (REI) files must start with `usbank`
+The first step to using this tool is obtaining the credit card transaction data.
+You can usually download a full year's worth of data at a time from each account.
 
-Currently `csv` and `xlsx/xls` files are supported.
+Once you obtain your data you can start using the UI.
 
-After putting your data into the raw directory, run the following script to ingest it:
+1. `Browse` and `Submit` buttons let you add the raw the data into the app's internal folder. Currently only `csv` files are supported.
+2. `Run Pipeline` button runs the data processing workflow that puts the data into a database.
```

Comment:
Currently if an `expenses.db` that was made with a previous version of the pipeline (and is incompatible for whatever reason), and it exists in the `data/` directory, then Run Pipeline fails. Likely because there is something somewhere in the code that tries to run on everything in the that directory.

I realize that that is probably out of scope for this PR, but wanted to flag it nonetheless.

Reply:
I see, that's another thing I didn't think about. What I think happened is you had old files in `data/raw` directory that are no longer valid. 

Via the UI only valid files reach the `data/raw` directory. We could modify the pipeline to only try to run on "valid" data files from the raw folder instead of taking everything and crashing.

Reply:
Specifically, I dropped `excel` files as a valid type since Amex's excel export didn't run out of the box due to the extra rows before the header.

Reply:
I updated the pipeline to ignore files it can't handle in the `data/raw` directory. Now it should work even if there are old files in there.